### ISSUE TITLE
[BUG] Engines off during flight

### ISSUE THREAD

**Issue by kferrano** [flybywiresim/a32nx#1242]
**Mod Version**

vmaster

**Describe the bug**

Engines go out during the flight for no reason, as do the battery and can no longer be switched on because the APU can no longer be started due to the failed battery.

**To Reproduce**

1. I can't describe it as this error occurs randomly during flight for no apparent reason
2.
3.

**Expected behavior**

That that won't happen?

**Actual behavior**

See **Describe the bug**

**References**

https://www.twitch.tv/videos/767073998

**Additional context**

Was this working before/when did the issue start occurring?
yes
Is this a problem in the vanilla unmodded game?

Discord username (if different from GitHub):

**Comment by wpine215** [comment]
Just to confirm, have you followed all of the steps in the release information (especially disabling all other traffic)?:
```
We recommend disabling multiplayer, AI traffic, and ground AI when flying the A320neo. This will help you avoid issues     introduced in previous MSFS patches where other aircraft loading in will cause switches and knobs to change and other issues. Furthermore, please remove the highonsnow FPS mod, it is no longer necessary.

Also be aware that some custom liveries may interfere with the A320neo. Using the Navigraph beta may also lead to issues.

We recommend running MSFS on Windows 10 version 2004.
```

**Comment by kferrano** [comment]
Yes, I don't use the AI or liveries

**Comment by wpine215** [comment]
Can you send a screenshot of your current traffic settings? And can you list the other non-aircraft/non-scenery mods you are using with the sim, if any?

**Comment by kferrano** [comment]
I can't take a screenshot because I'm flying. But multiplayer is off, so on "Group only" and AI traffic is off, I only use live weather

I don't use other mods.

**Comment by Benjozork** [comment]
> **Mod Version**
>
> vmaster

Could you provide the contents of `build-info.txt` ?

**Comment by CAN-KKATOU** [comment]
I have encountered this bug too. I think this is not the reason for this patch because I have deleted this patch and run the game and  
this error occurs again.

**Comment by SeanHanway** [comment]
I've also started to enounter this bug recently with no AI or mulitplayer traffic. I can't confirm if it happens on the unmodded A320 but my first time encountering it was just after I started using the A32NX Downloader to download the master branch and have had it happen a few times since. Before I used to manually pull the master branch straight from GitHub. I'd say it's just a coincidence but still worth mentioning.
